application/Dockerfile

```
@@ -9,7 +9,7 @@ RUN python -m pip install --upgrade --no-cache-dir \
     pip>=20.1.1 \
     setuptools>=49.1.0 \
     wheel>=0.34.2
-RUN python -m pip wheel -w /wheels .
+RUN python -m pip wheel -w /wheels -r requirements.txt
```

```suggestion
RUN python -m pip wheel -w /wheels -r requirements.txt
RUN python -m pip wheel -w /wheels .
```

I believe we still need to instruct `pip` to install the local package. It's possible we could do this all in one invocation, or by adding `.` to the `requirements.txt` file, but I think two separate invocations is the clearest way to express the intent.

I think this is the only remaining detail. Otherwise, this PR is looking great!

---

Ah yes, thanks for pointing this out